docs/source/models/resnet.rst

```diff
@@ -10,7 +10,7 @@ The ResNet model is based on the `Deep Residual Learning for Image Recognition
 Model builders
 --------------
 
-The following model builders can be used to instanciate a ResNet model, with or
+The following model builders can be used to instantiate a ResNet model, with or
```

Comment:
No opinion on this but I believe instanciate is correct as well

Reply:
I think "instanciate" is a misspelling of "instantiate" from https://www.wordsense.eu/instanciate/